app/jobs/people/external_invoice_payed_job.rb

```diff
@@ -0,0 +1,83 @@
+class People::ExternalInvoicePayedJob < BaseJob
+  def initialize(person, link, year)
+    @person = person
+    @link = link
+    @year = year
+
+    context = Invoices::SacMemberships::Context.new(Time.zone.today)
+    @member ||= Invoices::SacMemberships::Member.new(person, context)
+  end
+
+  def perform
+    if link_to_stammsektion?
+      update_role_dates
+    elsif link_to_neuanmeldung_stammsektion?
+      @person.confirmed_at = Time.zone.now if @person.confirmed_at.blank?
```

Comment:
würde ich in eine methode extrahieren, damit wir auf der gleiche erzählebene bleiben